auth-server/src/controllers/eventsHook.ts

```diff
@@ -0,0 +1,85 @@
+import { Request, Response } from 'express';
+
+import PostSubscription from '../model/PostSubscription';
+import { sendPostSubscriptionMail } from '../services/email';
+import getUserFromUserId from '../utils/getUserFromUserId';
+import messages from '../utils/messages';
+
+export const postCreateHook = async (req: Request, res: Response) => {
+	if (process.env.HASURA_EVENT_SECRET !== req.headers.hasura_event_secret) {
+		return res.status(403).json({ message: messages.UNAUTHORISED });
+	}
+
+	const post = req.body?.event?.data?.new || {};
+
+	const { id, author_id } = post;
+
+	if (!id || !author_id) {
+		console.error('[postAuthorSubscriptionHook] author_id not found', JSON.stringify(req.body, null, 2));
+		// sending success so that hasura don't retries
+		// retry will not help here
+		return res.json({ status: messages.SUCCESS });
```

Comment:
message.SUCCESS is probably very general. We should have a dedicated message here.